src/GitVersion.Core/VersionCalculation/BaseVersionCalculators/VersionStrategyBase.cs

```diff
@@ -2,11 +2,13 @@
 
 namespace GitVersion.VersionCalculation;
 
-public class VersionStrategyBase : IVersionStrategy
+public abstract class VersionStrategyBase : IVersionStrategy
```

Comment:
Marking `VersionStrategyBase` as `abstract` is probably a good idea, yes. 👍🏼

Reply:
Great that you also comment things which you like. This is also feedback which I appreciate